python_docs_theme/__init__.py

```diff
@@ -1,15 +1,22 @@
 from __future__ import annotations
 
 import hashlib
-import os
 from functools import cache
 from pathlib import Path
-from typing import Any
+
+TYPE_CHECKING = False
+if TYPE_CHECKING:
+    from typing import Any
+
+    from sphinx.application import Sphinx
+    from sphinx.util.typing import ExtensionMetadata
 
 import sphinx.application
 from sphinx.builders.html import StandaloneHTMLBuilder
```

Comment:
Lint fix: `E402 Module level import not at top of file`
```suggestion
import sphinx.application
from sphinx.builders.html import StandaloneHTMLBuilder

TYPE_CHECKING = False
if TYPE_CHECKING:
    from typing import Any

    from sphinx.application import Sphinx
    from sphinx.util.typing import ExtensionMetadata
```